shared/core/src/wasmJsMain/kotlin/com/shub39/grit/core/settings/presentation/ui/component/LocalePickerSheet.wasmJs.kt

```
@@ -0,0 +1,22 @@
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+package com.shub39.grit.core.settings.presentation.ui.component
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+
+@Composable actual fun LocalePickerSheet(onDismissRequest: () -> Unit, modifier: Modifier) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)